runtime/nls/j9vm/j9vm_ca.nls

```diff
@@ -524,14 +524,14 @@ J9NLS_VM_UNRECOGNIZED_XJNI_OPTION.user_response=Correct the option specified wit
 # END NON-TRANSLATABLE
 
 # help text for -Xjni:help
-J9NLS_VM_XJNI_OPTIONS_1=\u00das:\n
```

Comment:
Removing the `\n` here and in a number of other translations is going to change the formatting of the output. I don;'t think they should be removed.